src/kinect/updater.cpp

```diff
@@ -111,26 +148,43 @@ std::vector<EntityUpdate> mergeOverlappingXYConvexHulls(const rgbd::Image& image
       const EntityUpdate& u2 = updates[j];
 
       // If we collide, update the i convex hull
-      if (ed::convex_hull::collide(u1.chull, u1.pose_map.t, u2.chull, u2.pose_map.t, 0, 1e6))
+      if (ed::convex_hull::collide(u1.chull, u1.pose_map.t, u2.chull, u2.pose_map.t, 0, 1e6))  // This should prevent multiple entities above each other;1e6 is ok, because objects in other areas are ignored.
+//      if (ed::convex_hull::collide(u1.chull, u1.pose_map.t, u2.chull, u2.pose_map.t, 0, 0.0))  // This way, we get multiple entities above each other
```

Comment:
Remove commented code.
Where does the 1e6 come from?

Reply:
It has always been there